testing/telegraf/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Katie Holly <holly@fuslvz.ws>
+# Maintainer: Katie Holly <holly@fuslvz.ws>
+pkgname=telegraf
+pkgver=1.5.1
+pkgrel=0
+pkgdesc="A plugin-driven server agent for collecting & reporting metrics, part of the InfluxDB project"
+url="https://www.influxdata.com/time-series-platform/telegraf/"
+arch="x86_64 x86"
+license="MIT"
+makedepends="go"
+options="!strip"
+install=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/influxdata/$pkgname/archive/$pkgver.tar.gz
+	telegraf-makefile-ldflags.patch
+	telegraf.initd
+	"
+builddir="$srcdir"/$pkgname-$pkgver
+_godir="$srcdir"/go
+_vendor=influxdata
+
+build() {
+	mkdir -p "$_godir"/bin "$_godir"/src/github.com/"$_vendor"
+	ln -sf "$builddir" "$_godir"/src/github.com/$_vendor/$pkgname
+	export GOPATH="$_godir"
+	export PATH="$PATH:$_godir"
+	cd "$_godir"/src/github.com/$_vendor/$pkgname
+	go get -d -v .
```

Comment:
This is not acceptable, it fetches sources from Internet without any verification!

@ncopa, please revert this commit.

Reply:
@jirutka Numerous other packages do a similar thing, for example:

```
mylg-0.2.6.tar.gz: OK
>>> mylg: Unpacking /var/cache/distfiles/mylg-0.2.6.tar.gz...
+ cd /home/buildenv/aports/testing/mylg/src/mylg-0.2.6
+ export GOPATH=/home/buildenv/aports/testing/mylg/src
+ go get -v -d
github.com/briandowns/spinner (download)
github.com/fatih/color (download)
github.com/mehrdadrad/mylg (download)
github.com/chzyer/readline (download)
github.com/olekukonko/tablewriter (download)
github.com/mattn/go-runewidth (download)
Fetching https://golang.org/x/net/icmp?go-get=1
Parsing meta tags from https://golang.org/x/net/icmp?go-get=1 (status code 200)
get "golang.org/x/net/icmp": found meta tag get.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/icmp?go-get=1
get "golang.org/x/net/icmp": verifying non-authoritative meta tag
Fetching https://golang.org/x/net?go-get=1
Parsing meta tags from https://golang.org/x/net?go-get=1 (status code 200)
golang.org/x/net (download)
Fetching https://golang.org/x/net/ipv4?go-get=1
Parsing meta tags from https://golang.org/x/net/ipv4?go-get=1 (status code 200)
get "golang.org/x/net/ipv4": found meta tag get.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/ipv4?go-get=1
get "golang.org/x/net/ipv4": verifying non-authoritative meta tag
Fetching https://golang.org/x/net/ipv6?go-get=1
Parsing meta tags from https://golang.org/x/net/ipv6?go-get=1 (status code 200)
get "golang.org/x/net/ipv6": found meta tag get.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at https://golang.org/x/net/ipv6?go-get=1
get "golang.org/x/net/ipv6": verifying non-authoritative meta tag
github.com/gizak/termui (download)
github.com/maruel/panicparse (download)
github.com/mitchellh/go-wordwrap (download)
github.com/nsf/termbox-go (download)
github.com/k-sone/snmpgo (download)
github.com/geoffgarside/ber (download)
github.com/miekg/dns (download)
github.com/google/gopacket (download)
github.com/gorilla/mux (download)
github.com/rakyll/statik (download)
+ go build mylg.go
```

That's how I learned it from other packages but if there is a better way of handling go dependencies, please let me know with constructive criticism (you can reach out to me on Freenode IRC, my nick is Fusl and I'm in #alpine-devel and #alpine-linux) and I will change the code accordingly but don't simply say "this is wrong, I don't like this, please revert.".

Reply:
Oh gosh, someone did bad code review. If the project doesn't have bundled dependencies, then use e.g. Glide. There are already several packages using glide.

Reply:
Thank you for noticing that there are more aports calling `go get` in testing. I’ve reverted them too. 

Unfortunately it still may be hidden in Makefile, so the only way how to prevent it is to disconnect networking after fetch phase.

Reply:
If you use `abuild rootbld` it will disconnect networking in the build environment, this is a good way to test if there is anything bad happening.

Reply:
@kaniini I know. I’m considering if we can enable it on Travis.